cmd/control/control.go

```diff
@@ -80,6 +80,13 @@ func main() {
 	go func() {
 		errs <- database.DownsampleOverTime(conf.Database.DownsampleInterval, db)
 	}()
+	go func() {
+		c := database.SSHConfig{
```

Comment:
We already do a load of work to get an ssh key for onboarding, we should just use that.

https://github.com/gpuctl/gpuctl/blob/71ad89d9e52adfd0a5adb1a979e1de8e88021cbb/cmd/control/control.go#L51-L60

This will require changing `MonitorForDeadMachines` to take an `ssh.Signer` (& friends), but that's a good change.